### ISSUE TITLE
WebAPI Security Login Failed

### ISSUE THREAD

**Issue by tathagatabhattacharjee** [OHDSI/WebAPI#2260]
Trying to implement WebAPI security on ATLAS. Getting the message ‘Login Failed’ without showing the login prompts. Done all steps as suggested and recommended to setup the security, updated the settings.xml and config-local.js. Also made the necessary tables and inserts. It looks like that the url mentioned in config-local.js (user/login/db) is not available in WebAPI, so how to get this.

![Screenshot from 2023-04-27 14-49-23](https://user-images.githubusercontent.com/92773532/234829377-1de952f2-ac9e-4906-bd1a-e217d0623e6c.png)

**Comment by gkowalski** [comment]
Looks like your using 2.X instead of the 3.X . We went to 3.x and are having the same issue 
I'm looking at increasing the logging within the WebAPI program to see what its sending to our Active directory.

**Comment by konstjar** [comment]
It looks for me, you have a wrong configuration in config-local.js file. I do not see login form for username and password on your screen.

**Comment by tathagatabhattacharjee** [comment]
> 

Yes, precisely that is the problem, I could not understand why I was not able to see the login form

**Comment by tathagatabhattacharjee** [comment]
> Looks like your using 2.X instead of the 3.X . We went to 3.x and are having the same issue I'm looking at increasing the logging within the WebAPI program to see what its sending to our Active directory.

Where can I get 3.x, can you please the link

**Comment by konstjar** [comment]
I think you missed `"isUseCredentialsForm":true`

```
    {
      "name": "DB",
      "url": "user/login/db",
      "ajax": true,
      "icon": "fa fa-database",
      "isUseCredentialsForm":true
    }
